File: 코딩 기초 트레이닝/배열 비교하기-p181856/배열 비교하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181856

def solution(arr1, arr2):
    # arr2 -1
    # arr1 1
    # == 0

    if len(arr1) == len(arr2):
        if sum(arr1) == sum(arr2):
            return 0
        else:
            return 1 if sum(arr1) > sum(arr2) else -1
    else:
        return 1 if len(arr1) > len(arr2) else -1

File: 코딩 기초 트레이닝/배열 비교하기-p181856/배열 비교하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181856

function solution(arr1, arr2) {
    if(arr1.length === arr2.length) {
        let re1 = arr1.reduce((a, b) => a + b, 0);
        let re2 = arr2.reduce((a, b) => a + b, 0);

        if(re1 === re2) {
            return 0;
        } else {
            return re1 > re2 ? 1 : -1;
        }
    } else {
        return arr1.length > arr2.length ? 1 : -1
    }

    return answer;
}
